src/core/transaction.js

```diff
@@ -1112,6 +1112,77 @@ Transaction.prototype.setDeliverMin = function(deliver_min) {
   return this._setAmount('DeliverMin', deliver_min);
 };
 
+Transaction.prototype.suspendedPayment = function(options_) {
+  const options = _.merge({}, options_);
+
+  if (_.isUndefined(options.cancel_after)) {
```

Comment:
Right, but there's never a need for `if` statements if it's always going to convert from camel to snake at one point.